modules/javafx.graphics/src/main/java/javafx/scene/text/Text.java

```diff
@@ -1821,6 +1865,35 @@ final BooleanProperty caretBiasProperty() {
             }
             return caretBias;
         }
+
+        private IntegerProperty tabSize;
+
+        final int getTabSize() {
+            return tabSize == null ? TextLayout.DEFAULT_TAB_SIZE : tabSize.get();
+        }
+
+        final IntegerProperty tabSizeProperty() {
+            if (tabSize == null) {
+                tabSize = new StyleableIntegerProperty(TextLayout.DEFAULT_TAB_SIZE) {
+                    @Override public Object getBean() { return Text.this; }
+                    @Override public String getName() { return "tabSize"; }
+                    @Override public CssMetaData getCssMetaData() {
+                        return StyleableProperties.TAB_SIZE;
+                    }
+                    @Override protected void invalidated() {
```

Comment:
I think that annotations tend to go on a line above what they annotate,

Reply:
I'm following the pattern established elsewhere in the file for other property overrides. (underline, strikethrough, textAlignment, selectionEnd, etc.).  I would keep this for consistency and compactness, unless there are more votes against it.